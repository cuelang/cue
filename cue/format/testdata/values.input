a: 0e+1
a: 0e1
a: 0E+1
a: 0E1
a: .3e+1
a: .3E+1
