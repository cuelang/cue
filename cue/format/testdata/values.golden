a: 0e+1
a: 0e1
a: 0e+1
a: 0e1
a: 0.3e+1
a: 0.3e+1
a: 0.3
a: 3.0
a: 3.0T
a: 3.0e100

s: """
	x\"\"\"
	"""
